internal/notification/providers/chat/provider.go

```diff
@@ -0,0 +1,75 @@
+package chat
+
+import (
+	"bytes"
+	"encoding/json"
+	caos_errs "github.com/caos/zitadel/internal/errors"
+	"github.com/caos/zitadel/internal/notification/providers"
+	"net/http"
+	"net/url"
+	"unicode/utf8"
+)
+
+type Chat struct {
+	URL        *url.URL
+	SplitCount int
+}
+
+func InitChatProvider(config *ChatConfig) (*Chat, error) {
```

Comment:
i'd remove the pointers in the init funcs as configs are just structs in systemDefaults anyway